source/grid/tria.cc

```diff
@@ -6819,12 +6819,12 @@ namespace internal
                               // TODO: add comment
                               static const std::
                                 array<std::array<unsigned int, 3>, 6>
-                                  table = {{{{1, 0, 2}}, // 0
-                                            {{0, 1, 2}},
-                                            {{0, 2, 1}}, // 2
-                                            {{1, 2, 0}},
-                                            {{2, 1, 0}}, // 4
-                                            {{2, 0, 1}}}};
+                                  table = {{{{0, 1, 2}}, // 0
+                                            {{1, 0, 2}},
+                                            {{1, 2, 0}}, // 2
+                                            {{0, 2, 1}},
+                                            {{2, 0, 1}}, // 4
+                                            {{2, 1, 0}}}};
```

Comment:
Would this be a candidate for something that could also move into `ReferenceCell`? I *think* that what it encodes is the order of lines in faces of hexes.

Reply:
I have no idea what this table does. It isn't the lines of a triangle (that's the table in `standard_to_real_face_line()`).